### ISSUE TITLE
Accordion component doesn't work with screen reader

### ISSUE THREAD

**Issue by edcourtney74** [akveo/nebular#3057]
### Issue type

**I'm submitting a ...**  (check one with "x")

* [ x] bug report
* [ ] feature request

### Issue description

**Current behavior:**

The accordion component does not announce correctly to the user with JAWS screen reader. I believe this is because JAWS does not detect a <button> in the way the Nebular component is constructed.
**Expected behavior:**

The user should be notified if the accordion is expanded or collapsed and also instructed to press enter to activate the accordion. A correct example can be seen in the Bootstrap accordion.
**Steps to reproduce:**

Nebular accordion component example that does not announce correctly: https://akveo.github.io/nebular/docs/components/accordion/overview#nbaccordioncomponent

Bootstrap accordion component that does announce correctly:
https://getbootstrap.com/docs/5.0/components/accordion/

**Related code:**

```
insert short code snippets here
```

### Other information:
Using Nebular/theme 7.0.0

**npm, node, OS, Browser**
```
<!--
Node, npm: `node --version` and `npm --version`
OS: Windows (7/8/10). Linux (incl. distribution). macOS (El Capitan? Sierra?)
Browser: Chrome/Safari/Firefox/etc?
-->
```

**Angular, Nebular**
```
<!--
Check your `package-lock.json` or locate a `package.json` in the `node_modules` folder.
-->
```
